lib/libzrepl/data_conn.c

```diff
@@ -894,12 +894,21 @@ uzfs_zvol_rebuild_dw_replica(void *arg)
 		close(sfd);
 	}
 
-	if (wquiesce)
-		uzfs_zinfo_destroy_internal_clone(zinfo);
+#ifdef	DEBUG
+	if (inject_error.delay.rebuild_complete == 1)
+		sleep(10);
+#endif
+
+	if (wquiesce) {
+		if (uzfs_zinfo_destroy_internal_clone(zinfo)) {
```

Comment:
in else, better to set the stale_clone_exist to 0

Reply:
ok.